content/docs/user-guide/external-dependencies.md

```diff
@@ -27,20 +28,19 @@ supported:
 - HTTP
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
-
-In order to specify an external <abbr>dependency</abbr> for your stage, use the
-usual `-d` option in `dvc run` with the external path or URL to your desired
-file or directory.
+> Note [remote storage](/doc/command-reference/remote) is a separate feature.
```

Comment:
why is it needed?

Reply:
Because we mention remotes before so it may be confusing. Is a short note by the end of the section too much?